src/components/TodoLIst/TodoLIst.tsx

```diff
@@ -0,0 +1,62 @@
+import { useContext } from 'react';
+import cn from 'classnames';
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+import { TodosContext } from '../TodosContext/TodosContext';
+
+type Props = {
+  todos?: Todo[];
+};
+
+export const TodoList: React.FC<Props> = ({ todos }) => {
+  const { tempTodo } = useContext(TodosContext);
+
+  return (
+    <section className="todoapp__main" data-cy="TodoList">
```

Comment:
Use semantic tags for list `ul` `li`. Don't forget to remove default `li` dots and `ul` paddings